### DISCUSSION TITLE
How to install `vscode-html-language-server` on Windows 11?

### DISCUSSION THREAD

**Discussion by mizzunet** [helix-editor/helix#9464] [Q&A]
From where do I get this `vscode-html-language-server` executable? I have scoop package manager and `node` installed.

I tried `npm install vscode-html-languageservice`. Still don't work.

Thanks

**Comment by kirawi** [comment]
https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers#html

**Comment by cheneysan** [comment]
`npm i -g vscode-langservers-extracted` worked for me

Seems that it installs:
`vscode-html-language-server`
`vscode-css-language-server`
`vscode-json-language-server`
